gmail_api/src/main/java/com/android/GetGmailPINBasedOnSubstring.java

```diff
@@ -0,0 +1,138 @@
+package com.android;
+
+
+
+import com.testsigma.sdk.AndroidAction;
+import com.testsigma.sdk.ApplicationType;
+import com.testsigma.sdk.Result;
+import com.testsigma.sdk.WebAction;
+import com.testsigma.sdk.annotation.Action;
+import com.testsigma.sdk.annotation.RunTimeData;
+import com.testsigma.sdk.annotation.TestData;
+import lombok.Data;
+
+import javax.mail.*;
+import java.util.Properties;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@Data
+@Action(actionText = "Get value from gmail using EmailID and Password and trim the value after testdata upto nth digits and store the value in a runtime variable var1",
```

Comment:
Change the Action text,
"Get value from gmail using Email: EmailID and Key: Password and trim the value after data testdata upto nth digits and store the value in a runtime variable"